### DISCUSSION TITLE
Polish translation

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#292] [Q&A] [answered]
@Ciutnik Thanks for the impressive work! :+1: 

Polish translation will be available in next Cruiser.

---

Polish voice navigation is already available.

**Comment by Ciutnik** [comment]
Nothing more nothing less. After the last exchange of comments, we can consider the work on the translation successful and complete (as of today) 🇵🇱

Minor remarks regarding the original English nomenclature will be discussed sometime in the forum 🙂

Thank you @devemux86 for patience and understanding 👍

**Comment by airald** [comment]
Sometimes is difficult set a name for the different features. When translating, some terms can not be literally translated. As far as I know, polish is one of the most difficult or complex languages in the world.

**Comment by devemux86** [comment, marked as answer]
Polish translation and voice navigation are available in [Cruiser 3.0.32](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

@Ciutnik thanks for your help! :+1:
